tests/php/Integration/Api/ApiFacadeTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhelTest\Integration\Api;
+
+use Gacela\Framework\Bootstrap\GacelaConfig;
+use Gacela\Framework\Gacela;
+use Phel\Api\ApiFacade;
+use PHPUnit\Framework\TestCase;
+
+final class ApiFacadeTest extends TestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        Gacela::bootstrap(__DIR__, GacelaConfig::withPhpConfigDefault());
+    }
+
+    public function test_number_of_grouped_functions(): void
+    {
+        // This test in isolation works, but when running with all other integration tests fails,
```

Comment:
🤔

Reply:
I spent some time trying to avoid this comment... But I couldn't make it work after an hour. This is clearly an smell that should be improved, but I didn't see it as a blocker for the all concept/PR. I will try to make it better in a future iteration. Anyway, feel free to try it on your own if you feel brave enough hehe